lib-gdal-preferences-ui/pom.xml

```diff
@@ -33,6 +33,7 @@
         <dependency>
             <groupId>org.esa.snap</groupId>
             <artifactId>lib-gdal</artifactId>
+            <version>${project.parent.version}</version>
```

Comment:
better replace by ${snap.version} to be consistent with usage in other modules.

Reply:
Updates made.